pkg/controller/performanceprofile/components/tuned/tuned.go

```diff
@@ -8,7 +8,7 @@ import (
 	"strings"
 	"text/template"
 
-	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1alpha1"
+	performancev1alpha1 "github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
```

Comment:
same (and for all other files in this package)

Reply:
ok